lib/python/picongpu/extra/plugins/data/radiation.py

```diff
@@ -127,3 +128,40 @@ def get_vector_n(self):
     def get_timestep(self):
         """return PIC iteration (timestep) at which the data was produced (unit: PIC-cycles)"""
         return self.iteration
+
+    def get_distributedAmplitude(self):
+        """
+        Return a 4D complex array of all distributed amplitudes
+        Array dimensions:
+        0: index GPU/worker
+        1: direction
+        2: frequency
+        3: polarization [x, y, z]
+        returns None if no distributed amplitudes are available
+        """
+        try:
+            h_distAmp = self.iteration.meshes["Amplitude_distributed"]
+        except IndexError:
+            warnings.warn("Warning: no distributed amplitude available.")
+            return None
+
+        # get shape of distributed amplitude
+        shapeDistAmp = h_distAmp["x"].shape
+        # add 3 vector components
+        shapeDistAmp.append(3)
```

Comment:
What's the magic number `3`? Maybe a named constant would be nice.

Reply:
3 is for the 3 spacial dimensions 
Since there is no `const uint` in python, I personally favor using literals instead. 
What would you prefer here to make clear that is a *never changing* value?

Reply:
@chillenzer I used a class variable to make the global scope of that 3 clear. Is this in agreement with what you wanted?